File: lab/python/POO/circulo.py
class Circulo:
    def __init__(self, raio) -> None:
        self.raio = raio
        self.pi = 3.141

    def area(self):
        return self.pi * self.raio**2

    def perimetro(self):
        return 2 * self.pi * self.raio


c = Circulo(3)
print(f"Área = {c.area():.2f}, Perímetro = {c.perimetro():.2f}")

File: lab/javascript/POO/circulo.js
class Circulo{
    constructor(raio){
        this.raio = raio;
        this.pi = 3.141;
    }

    area(){
        return this.pi * this.raio**2;
    }

    perimetro(){
        return 2 * this.pi * this.raio;
    }
}


c = new Circulo(3);
console.log(`Área = ${c.area().toFixed(2)}, Perímetro = ${c.perimetro().toFixed(2)}`)
